src/plugindata.h

```diff
@@ -183,6 +183,11 @@ typedef struct GeanyData
 	struct GeanyTemplatePrefs	*template_prefs;	/**< Template settings */
 	gpointer					*_compat;			/* Remove field on next ABI break (abi-todo) */
 	GSList						*filetypes_by_title; /**< See filetypes.h#filetypes_by_title. */
+	/** Global object signalling events via signals
+	 *
+	 * This is mostly for language bindings. Otherwise prefer to use
+	 * plugin_signal_connect() instead this which adds automatic cleanup. */
+	GeanyObject * const			object;
```

Comment:
Also, it should probably be `@gironly` as I can't see a real use case in plain C that wouldn't be better served by the `plugin_signal_connect()` API.